Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,19 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// Task1: Display names of people in Gryffindor
+console.log("Gryffindor Members:");
+hogwarts.forEach(({ firstName, lastName, house }) => {
+  if (house === "Gryffindor") {
+    console.log(`${firstName} ${lastName}`);
+  }
+});
+
+// Task2: Display names of teachers who have pets
+console.log("\nTeachers with Pets:");
+hogwarts.forEach(({ firstName, lastName, occupation, pet }) => {
```

Comment:
Your code works well 👍
Non-blocking suggestion: If you wanted to improve clarity, you can use `.filter` to only include records where occupation === 'Teacher' && pet before running `.forEach`.